smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/Naming.kt

```diff
@@ -25,15 +25,25 @@ import java.util.logging.Logger
 fun Shape.defaultName(serviceShape: ServiceShape): String = id.getName(serviceShape).toPascalCase()
 
 /**
- * Get the default name for a member shape (for code generation)
+ * Get the default name for a member shape (for code generation), e.g., "bucketName"
  */
 fun MemberShape.defaultName(): String = memberName.toCamelCase()
 
 /**
- * Get the default name for an operation shape
+ * Get the default name for a member shape (for code generation), e.g., "BucketName"
+ */
+fun MemberShape.capitalizedDefaultName(): String = defaultName().replaceFirstChar(Char::uppercaseChar)
```

Comment:
I understand why this is added but I hate this. I miss `defaultName().capitalize()` ... sigh